src/v2/Apps/ViewingRoom/__tests__/ViewingRoomApp.jest.tsx

```diff
@@ -375,6 +376,7 @@ describe("ViewingRoomApp", () => {
       expect(wrapper.find("ViewingRoomTabBar").length).toBe(0)
       expect(wrapper.find("ViewingRoomContentNotAccessible").length).toBe(0)
       expect(wrapper.html()).not.toContain("some child")
+      jest.runAllTimers()
       expect(mediator.trigger).toHaveBeenCalled()
```

Comment:
Unrelated, but I guess this spec started failing. There's a `setTimeout` wrapping the call to this function on the next tick to deal with a race condition.